stdlib/json.gr

```diff
@@ -417,15 +417,9 @@ provide enum FormattingChoices {
   },
 }
 
-record JsonWriterCompactImplHelper {
-  buffer: Buffer.Buffer,
-  emitEscapedQuotedString: String => Void,
-}
-
-record JsonWriterPrettyImplHelper {
+record JsonWriterImplHelper {
```

Comment:
Is this just a `JsonWriter` now? Why still name it `ImplHelper` at all?

Reply:
From what i understand `JSONWriter` is juts `emit` for the future purpose of being able to expose it for reuse, this is still a behind the scenes `Impl` used by a given `JSONWriter`.

Reply:
The only reason for the ImplHelper suffix was to emphasize that it's just an implementation detail and not something to expose at the API level.